src/current/v23.2/select-for-update.md

```diff
@@ -16,22 +16,50 @@ The following diagram shows the supported syntax for the optional `FOR` locking
 {% remote_include https://raw.githubusercontent.com/cockroachdb/generated-diagrams/{{ page.release_info.crdb_branch_name }}/grammar_svg/for_locking.html %}
 </div>
 
+{{site.data.alerts.callout_success}}
 For the full `SELECT` statement syntax documentation, see [Selection Queries]({% link {{ page.version.version }}/selection-queries.md %}).
+{{site.data.alerts.end}}
 
 ## Parameters
 
-### Locking strengths
+|            Parameter             |                                                                                                                                                      Description                                                                                                                                                      |
+|----------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `FOR SHARE`/`FOR KEY SHARE`      | <span class="version-tag">New in v23.2:</span> Acquire a shared lock on the rows returned by the [`SELECT`]({% link {{ page.version.version }}/selection-queries.md %}) statement. Shared locks are not enabled by default for `SERIALIZABLE` transactions. For details, see [`FOR SHARE` usage](#for-share-usage).
+| `FOR UPDATE`/`FOR NO KEY UPDATE` | Acquire an exclusive lock on the rows returned by the [`SELECT`]({% link {{ page.version.version }}/selection-queries.md %}) statement. For details, see [`FOR UPDATE` usage](#for-update-usage).  
```

Comment:
nit: It's true that right now `FOR KEY SHARE` and `FOR NO KEY UPDATE` are aliased to the other strengths, but I find it a little confusing to list them together. You might consider giving them their own rows with Descriptions like "not implemented, currently aliased to `FOR SHARE`" or something.

Reply:
Oh, I just realized this is how the docs were before. Well, maybe ignore me then. 🙂

Reply:
Added these to an Aliases section!